### ISSUE TITLE
Growl: FacesSeverity not read out by screen readers

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#5153]
# Summary
There is a nice icon that represents FacesSeverity but there is no alternative text for it. I think primefaces should provide message keys per severity for i18n as well.

## 1) Environment
* PrimeFaces version: 7.0.6
* PrimeFaces showcase: https://www.primefaces.org/showcase/ui/message/growl.xhtml

## 2) Expected behavior
FacesSeverity is read in addition to message text.

## 3) Actual behavior
FacesSeverity alternative text is missing

## 4) Steps to reproduce
Produce a growl message and listen to the output of a screen reader.

## 5) Sample HTML from showcase
`<span class="ui-growl-image ui-growl-image-error"></span>`
`<div class="ui-growl-message">`
`<span class="ui-growl-title">Message: Validation Error: Value is required.</span><p>Message: Validation Error: Value is required.</p>`
`</div>`

## 6) Sample Fix
`<span class="ui-growl-image ui-growl-image-error"></span>`
`<span class="ui-helper-hidden-accessible">error</span>`
`<div class="ui-growl-message">`
`<span class="ui-growl-title">Message: Validation Error: Value is required.</span><p>Message: Validation Error: Value is required.</p>`
`</div>`

**Comment by melloware** [comment]
Couldn't we add a tooltip to the icon or an aria-label on the icon to give the screenreader a hint rather than adding another SPAN in the HTML?

**Comment by MarkusChur** [comment]
I checked that but then the text is not read out. I guess things get complicated when using aria-live regions. When adding another `<span>` the text is read out correctly and it does not break the visual appearance

**Comment by melloware** [comment]
Please review my PR.

**Comment by MarkusChur** [comment]
done
